### DISCUSSION TITLE
How to use range on table joins.

### DISCUSSION THREAD

**Discussion by Tutuldot** [supabase/supabase#16549] [Questions] [answered]
Hi,

Good day. I have the code below that return the data that i need. My problem is i need to limit the data i have retrieved from jobdetails for pagination purposes. Is possible to add range on joined table? 

`   let { data, error, status } = await supabase
            .from('jobs')
            .select('*,jobdetails(id, created_at, modified_at, generatedCoverLetter, status, job_header, job_id, jobs_masterlist(source,source_site, url,details, jobid, jobname,jobverified,tags))')
            .eq('user_id', user?.id)
            .eq('id',params.id)`

**Comment by GaryAustin1** [comment, marked as answer]
For limit you can use foreign tables per the docs...  
![image](https://github.com/supabase/supabase/assets/54564956/52fcb4b3-aaa0-43ba-99fd-e20f975dd6ba)

The same thing seems to work for .range() even though not in the docs.

**Reply by Tutuldot** [reply]
thanks a lot. this is what i can looking for.
